packages/select/src/__tests__/examples.tsx

```diff
@@ -2,40 +2,33 @@ import type { DummyItem } from './constants';
 import type { MultiSelectTriggerProps } from '../MultiSelect';
 import type { SingleSelectTriggerProps } from '../SingleSelect';
 
-import { Tooltip } from '@launchpad-ui/tooltip';
+import { IconButton } from '@launchpad-ui/button';
+import { Edit } from '@launchpad-ui/icons';
+import { TagGroup, TagItem } from '@launchpad-ui/tag';
 
 const CustomMultiSelectTrigger = (props: MultiSelectTriggerProps<DummyItem>) => {
   const { state, triggerProps, triggerRef } = props;
 
+  console.log(state.selectedItems);
```

Comment:
Guessing we wanna remove this.

Reply:
These files crept in, shouldn't be in the PR. Thanks for flagging!